src/bauhaus/bauhaus.c

```diff
@@ -2427,6 +2437,8 @@ void dt_bauhaus_show_popup(GtkWidget *widget)
       darktable.bauhaus->change_active = 1;
       if(!d->entries->len) return;
       tmp.height = darktable.bauhaus->line_height * d->entries->len;
+      // if the popup is detached, we show the lable in any cases, in a special line
+      if(w->detached_popup) tmp.height += darktable.bauhaus->line_height;
```

Comment:
This moves the selected item down one line so that it no longer aligns with the mouse position. Using a shortcut to call up the popup and then immediately clicking would change to the "previous" item, rather than leaving the selection unchanged as used to be the behavior.

Reply:
You're right : we also need the move the popup and the selection of 1 line !  PR #12086 should fix that. Thanks for the report !